BackofficeBundle/Validator/Constraints/BlockNodePatternValidator.php

```diff
@@ -67,4 +71,22 @@ public function validate($node, Constraint $constraint)
             }
         }
     }
+    protected function getRefBlock(AreaContainerInterface $container)
+    {
+        $areas = $container->getAreas();
+        if (count($areas) > 0){
+            foreach ($areas as $area) {
+                $this->getRefBlock($area);
+            }
+        } else {
+            $blocks = $container->getBlocks();
+            if (count($blocks) > 0){
+                foreach ($blocks as $block) {
+                    if($block['nodeId'] !== NodeInterface::TRANSVERSE_NODE_ID) {
```

Comment:
You are just checking for the block saved in the current node, this way the check should be only on nodeId == 0.

With your test, if we add a block from another node, the check will also be done